phalcon/Mvc/Collection.zep

```diff
@@ -1013,55 +1021,6 @@ abstract class Collection implements EntityInterface, CollectionInterface, Injec
         let this->skipped = skip;
     }
 
-    /**
-     * Allows to perform a summatory group for a column in the collection
-     */
-    public static function summatory(string! field, conditions = null, finalize = null) -> array
```

Comment:
Why you decided to remove this method? Anyway, should we reflect this removal in the change log?